.changelog/20353.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+ui: adds V2CatalogEnabled to config that is passed to the ui
```

Comment:
https://github.com/hashicorp/consul/blob/main/.changelog/20353.txt
This file does exist, I think you maybe need to pull in main?